src/DynamicData/Cache/ObservableCacheEx.cs

```diff
@@ -4832,6 +4834,313 @@ sortOrder switch
         where TSource : notnull
         where TSourceKey : notnull => new TransformMany<TDestination, TDestinationKey, TSource, TSourceKey>(source, manySelector, keySelector).Run();
 
+    /// <summary>
```

Comment:
I wonder whether we should break this class into a series of partial classes as it's becoming huge.

Reply:
That's not a bad idea... But if we do it, it should be in dedicated PR.